schema-kenerator-validation-swagger/README.md

```diff
@@ -0,0 +1,31 @@
+# Schema-Kenerator Validation Swagger
+
+[![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.github.smiley4/schema-kenerator-validation-swagger/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.github.smiley4/schema-kenerator-validation-swagger)
```

Comment:
I don't know how this `badge.svg` file actually gets created... I'm assuming it's automatic, but if not, let me know what I need to change to make this bit work.

Reply:
tbh, i'm not 100% sure either right now, i think it gets created with the maven central release - we'll see with the release :)